libres/tests/analysis/test_update.cpp

```diff
@@ -46,16 +53,16 @@ struct model {
     int size() { return 2; }
 };
 
-SCENARIO("Running analysis update without row scaling on linear model",
+SCENARIO("Running analysis update with and without row scaling on linear model",
          "[analysis]") {
 
     GIVEN("Fixed prior and measurements") {
 
         analysis_module_type *enkf_module = analysis_module_alloc("STD_ENKF");
         analysis_module_set_var(enkf_module, ENKF_TRUNCATION_KEY_, "1.0");
-        rng_type *rng = rng_alloc(MZRAN, INIT_DEFAULT);
+        rng_type *rng = rng_alloc(MZRAN, INIT_DEV_URANDOM);
```

Comment:
I would stick to `INIT_DEFAULT` in order to get reproducible results

Reply:
The idea is to write the type of tests that are agnostic to the seed, i.e., test properties that should hold no matter the seed.

Reply:
I see the point - but you can not actually verify that you have succeeded in that; other than that flakyness would imply that the test is not agnostic to seed.

Reply:
After some thought, I agree with you and have switched back to INIT_DEFAULT.
Thanks!